backend/models/class.model.ts

```diff
@@ -0,0 +1,35 @@
+import mongoose, { Schema, Document, model } from "mongoose";
+
+export interface Class extends Document {
```

Comment:
could you add some documentation? just a line for each field and the purpose of the Class object (you can refer to `school.model.ts` as an example)